### PR TITLE
Prac 5 code review request

### PR DESCRIPTION
**Pull request by ellBasso**
Please check formatting, naming and logic @danniellejones, Thanks.

### CODE REVIEW

File: prac_05/word_occurrences.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+"""
+Write a program to count the occurrences of words in a string. The program should ask the user for a string,
+then print the counts of how many of each word are in the file.
+"""
+
+string_to_count = (input("Please enter a string to count: ")).lower().split(" ")
```
Comment (danniellejones) [inline prac_05/word_occurrences.py]: string_to_count could just be text like the prompt in the output? "Text: this is a collection of words..."

File: prac_05/word_occurrences.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+counted_words = {}
+for word in string_to_count:
+    if word in counted_words:
```
Comment (danniellejones) [inline prac_05/word_occurrences.py]: Use .get here?

File: prac_05/emails.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
+"""
+Write a program that stores users' emails (unique keys) and names (values) in a dictionary.
+Ask the user for their email until they enter a blank one.
+Use a separate function to get the name from the email
+"""
+
+
+def main():
+    email_directory = {}
+    email_address = input("Email: ")
```
Comment (danniellejones) [inline prac_05/emails.py]: Keep naming concise, email?

File: prac_05/emails.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
+"""
+Write a program that stores users' emails (unique keys) and names (values) in a dictionary.
+Ask the user for their email until they enter a blank one.
+Use a separate function to get the name from the email
+"""
+
+
+def main():
+    email_directory = {}
```
Comment (danniellejones) [inline prac_05/emails.py]: Name it closer to what goes in it? This dictionary is mapping... emails to names. So a good name might be email_to_name?

File: prac_05/emails.py [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+        name_confirm = input("Is your name {}? (Y/n) ".format(name))
+        if name_confirm.upper() != "Y" and name_confirm != "":
+            name = input("Name: ")
+        email_directory[email_address] = name
+        email_address = input("Email: ")
+    for address, name in email_directory.items():
+        print("{} ({})".format(name, address))
+
+
+def get_name(email_address):
```
Comment (danniellejones) [inline prac_05/emails.py]: get_name sounds like it is getting the users input for name?

File: prac_05/word_occurrences.py (hunk 2 above)
Comment (ellBasso) [reply prac_05/word_occurrences.py]: Good catch.
Thank you for all the feedback! :)

**Pull request merged**